project/ScalafixBuild.scala

```diff
@@ -67,6 +67,7 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         "scalameta" -> scalametaV,
         scalaVersion,
         "supportedScalaVersions" -> supportedScalaVersions,
+        "supportedScalaVersionsForExplicitResultTypeRule" -> supportedScalaVersionforExplicitResultRule,
```

Comment:
this is the build info of core, I don't think it should know anything about built-in rules